Practica real con tabla/ViewController.swift

```diff
@@ -43,49 +43,46 @@ class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSour
 
     func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
 
+        // use large numbers to avoid collisions with system-defined views
+        let messageViewTag = 2
+        let ballonViewTag = 4
+        let labelTag = 8
 
         var ballonview = UIImageView()
         var label = UILabel()
 
-        let cell:UITableViewCell? = tableView.dequeueReusableCellWithIdentifier("Cell", forIndexPath: indexPath)
+        let cell = tableView.dequeueReusableCellWithIdentifier("Cell", forIndexPath: indexPath)
```

Comment:
`tableView:dequeueReusableCellWithIdentifier:` will always return a `UITableViewCell`, which will not be `nil`.